src/db.rs

```diff
@@ -899,7 +899,7 @@ impl DB {
         Ok(())
     }
 
-    pub fn repair<P: AsRef<Path>>(opts: Options, path: P) -> Result<(), Error> {
```

Comment:
What is a point ? Do we need to use `Options` after repairing ?

Reply:
Yes, that's exactly what we do:
```rust
let db_corrupted = Path::new(path).join(Database::CORRUPTION_FILE_NAME);
if db_corrupted.exists() {
	DB::repair(&opts, path)?;
	//         ^^^^^
}
// ...
DB::open_cf(&opts, path, &cfnames)
```